backend/app/Http/Controllers/User/ShowIndividualUserController.php

```diff
@@ -28,7 +40,127 @@ public function __construct(Gate $gate)
+         * @property int $live_area_id
+         * @property int $grade
+         * @property int $gender
+         * @property string $birth_area
+         * @property string $live_area
+         * @property string $live_area
+         */
+        $user = DB::table('user_infos')
+            ->where('user_id', $user_id)
+            ->join('users', 'user_infos.user_id', '=', 'users.id')
+            ->leftJoin('u_u_majors', 'user_infos.u_u_major_id', '=', 'u_u_majors.id')
+            ->select('users.id', 'users.name', 'users.profile_photo_path', 'users.user_role_id', 'user_infos.status', 'user_infos.grade', 'user_infos.gender', 'user_infos.university_meta', 'user_infos.company_meta', 'user_infos.birth_area_id', 'user_infos.live_area_id', 'user_infos.group_affiliation', 'user_infos.is_publish_birth_day', 'user_infos.birth_day', 'user_infos.hobbies', 'user_infos.interests', 'user_infos.motto', 'user_infos.slack_name', 'user_infos.discord_name', 'user_infos.line_name', 'user_infos.github_id', 'u_u_majors.name as uu_major', 'u_u_majors.faculty_id as uu_faculty')
+            ->first();
+
+        $userAreas = Area::whereIn('id', [$user->birth_area_id, $user->live_area_id])->get();
+        // 在住と出身が同じだった場合、返り値1つなので
+        /** @extends  */
+        $userBirthArea = $userAreas->first(fn (Area $area) => $area->id === $user->birth_area_id);
+        $userLiveArea = $userAreas->first(fn (Area $area) => $area->id === $user->live_area_id);
+        // userにbirthとliveメソッドを追加
+        $user->birth_area = $userBirthArea->prefecture_code->label() . $userBirthArea->municipality;
+        $user->live_area = $userLiveArea->prefecture_code->label() . $userLiveArea->municipality;
+
+        $links = DB::table('user_links')
+            ->where('user_id', $user_id)
+            ->get();
+
+        $projects = DB::table('project_belongeds')
+            ->where('user_id', $user_id)
+            ->join('projects', 'project_belongeds.project_id', '=', 'projects.id')
+            ->get();
+
+        $events = UserTimeline::where('user_id', $user_id)
+            ->orderBy('start_date', 'asc')
+            ->get();
+
+        $relatedUsers = DB::table('user_infos')
+            ->where('grade', $user->grade)
+            ->join('users', 'user_infos.user_id', '=', 'users.id')
+            ->leftJoin('u_u_majors', 'user_infos.u_u_major_id', '=', 'u_u_majors.id')
+            ->select('users.name', 'users.profile_photo_path', 'user_infos.user_id', 'user_infos.grade', 'user_infos.u_u_major_id', 'user_infos.university_meta', 'user_infos.company_meta', 'u_u_majors.name as uu_major', 'u_u_majors.faculty_id as uu_faculty')
+            ->inRandomOrder()
+            ->limit(2)
+            ->get();
+
+        /**
+         * enumを用いて学部・学年・性別・国・都道府県・タイムラインのジャンル・関連ユーザを取得
+         */
+        $user->uu_faculty = $user->uu_faculty == null ? "" : UUFaculty::from($user->uu_faculty)->label();
+        $user->grade = Grade::from($user->grade)->label();
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property App\Models\UserInfo::$grade (App\Enums\Grade) does not accept string.